### ISSUE TITLE
Restore Focus to Change Location Button after User Presses ESC

### ISSUE THREAD

**Issue by aroubin** [LSCOIT/LegalNavigator#700]
_From @xilin95 on November 29, 2018 19:3_

**Issue:** Currently when user presses ESC to exit the map modal, the focus doesn't restore back to the change location button

**Reference:**
- **Accessibility standards on modals:** https://www.w3.org/TR/wai-aria-practices/#dialog_modal

_Copied from original issue: Microsoft/Access2Justice#852_

**Comment by KellyeDC** [comment]
Confirm issue resolved.

**Comment by KellyeDC-zz** [comment]
Is this a desired feature to complement geocoding changes @grawdon and @breeglej ?

**Comment by breeglej** [comment]
@KellyeDC @grawdon - This doesn't sound like something that needs to be done before pilot.  I'd put on hold.
